### ISSUE TITLE
How to specify JVM args per cluster instead upon invoking start method?

### ISSUE THREAD

**Issue by smiklosovic** [riptano/ccm#696]
Hi, @ptnapoleon 

I know there is jvm_args parameter for start method in Cluster class but it means that I have to specify this every time.

Cassandra's dtests are starting nodes but there is not simple way how to say "for all dtests, for every node I start via ccm, I want to set -Xmx and -Xms for each node to so and so value".

There is so many nodes in dtests it is just wrong to specify it every single time per node (not even sure thats possible).

Naive approach would be to introduce jvm_args as a paramter of __init__ on Cluster and if jvm_args in start method are None, it would take args from self.

**Comment by ptnapoleon** [comment]
I agree that this is a good feature, and I think the naive approach is reasonable.

**Comment by smiklosovic** [comment]
@ptnapoleon there are apparently these properties you can override the defaults

https://github.com/apache/cassandra/blob/a85196246c009566aa838156f3f56d00145e76ad/.circleci/config.yml#L84-L85

**Comment by ptnapoleon** [comment]
I was sure defaults were settable, I just wasn't able to find them. Thanks! This is how I've always seen the dtests configured.
